### ISSUE TITLE
The paper pictures on index page can be changed to some other things, e.g., news? new paper titles?

### ISSUE THREAD

**Issue by Zhiyi-Zhang** [UCLA-IRL/UCLA-IRL.github.io#2]
Because from the figure I can barely figure out what are papers about.
Maybe we could think of new ideas about how to update this section.

**Comment by jdellaverson19** [comment]
Yes, those are just a concept (links are not to current papers, images are
just screenshots, etc).
